### PR TITLE
Some more German translation

### PR DESCRIPTION
**Pull request by Pilzinsel64**

### CODE REVIEW

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 1]
```diff
@@ -218,6 +233,42 @@ enderio.phasedIron.name=Pulsierendes Eisen
...
 enderio.darkSteel.name=Dunkelstahl
 enderio.soularium.name=Soularium
+enderio.endSteel.name=End-Stahl
+enderio.endSteelNugget.name=End-Stahl-Nugget
+
+enderio.crudeSteel.name=Lehmzusammensetzung
+enderio.crystallineAlloy.name=Kristallin-Legierung
+enderio.melodicAlloy.name=Melodische Legierung
+enderio.stellarAlloy.name=Hervorragende Legierung
+enderio.crystallinePinkSlime.name=Melodischer Rose Schleim
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:243]:
```suggestion
enderio.crystallinePinkSlime.name=Melodischer Rosa Schleim
```

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 2]
```diff
@@ -218,6 +233,42 @@ enderio.phasedIron.name=Pulsierendes Eisen
...
+enderio.energeticSilver.name=Energetisches Silber
+enderio.vividAlloy.name=Lebendige Legierung
+
+enderio.crudeSteelGrindingBall.name=Mahlwerkkugel aus Lehmzusammensetzung
+enderio.crystallineAlloyGrindingBall.name=Mahlwerkkugel aus Kristallin-Legierung
+enderio.melodicAlloyGrindingBall.name=Mahlwerkkugel aus Melodischer Legierung
+enderio.stellarAlloyGrindingBall.name=Mahlwerkkugel aus Hervorragender Legierung
+enderio.crystallinePinkSlimeGrindingBall.name=Mahlwerkkugel aus Melodischem Rose Schleim
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:251]:
```suggestion
enderio.crystallinePinkSlimeGrindingBall.name=Mahlwerkkugel aus Melodischem Rosa Schleim
```

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 3]
```diff
@@ -403,6 +461,16 @@ tile.blockZombieGenerator.tooltip.detailed.line2=um Energie zu erzeugen
...
 
+tile.blockFrankenzombieGenerator.name=Frank'N'Zombie-Generator
+tile.blockFrankenzombieGenerator.tooltip.detailed.line1=Konsumiert Nahrhaftes Destillat,
+tile.blockFrankenzombieGenerator.tooltip.detailed.line2=um Energie zu erzeugen
+enderio.frankenzombieGenerator.fuelTank=Nahrhafter Tank
+
+tile.blockEnderGenerator.name=Ender-Generator
+tile.blockEnderGenerator.tooltip.detailed.line1=Konsumiert Tau des Voids,
+tile.blockEnderGenerator.tooltip.detailed.line2=um Energie zu erzeugen
+enderio.enderGenerator.fuelTank=Void-Tank
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:472]:
```suggestion
enderio.enderGenerator.fuelTank=Leere-Tank
```

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 4]
```diff
@@ -613,6 +681,12 @@ item.itemTravelStaff.tooltip.detailed.line4=<shift><r-click> wird den Spieler ei
 item.itemTravelStaff.tooltip.detailed.line5=kurze Distanz teleportieren
 enderio.itemTravelStaff.notEnoughPower=Nicht genügend Energie
 
+item.itemTeleportStaff.name=Stab der Teleportation
+item.itemTeleportStaff.tooltip.detailed.line1=Rechtsklick, um zu teleportieren, wohin auch immer du schaust
+item.itemTeleportStaff.tooltip.detailed.line2=Shift-Rechtsklick, um zu Reiseankern zu telportieren
+item.itemTeleportStaff.tooltip.detailed.line3=Rechtsklick auf einen benachbarten Blcok, um zu versuchen
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:687]:
```suggestion
item.itemTeleportStaff.tooltip.detailed.line3=Rechtsklick auf einen benachbarten Block, um zu versuchen
```

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 5]
```diff
@@ -702,46 +778,99 @@ item.darkSteel_pickaxe.name=Dunkelspitzhacke
 item.darkSteel_axe.name=Dunkelaxt
 item.darkSteel_shears.name=Dunkelschere
 
+#End Steel
+item.endSteel_helmet.name=Endstahl-Helm
+item.endSteel_chestplate.name=Endstahl-Brustpanzer
+item.endSteel_leggings.name=Endstahl-Beinschutz
+item.endSteel_boots.name=Endstahl-Steifel
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:785]:
```suggestion
item.endSteel_boots.name=Endstahl-Stiefel
```

File: src/main/resources/assets/enderio/lang/de_DE.lang [hunk 6]
```diff
@@ -702,46 +778,99 @@ item.darkSteel_pickaxe.name=Dunkelspitzhacke
...
+item.endSteel_sword.name=Endstahl-Schwert
+item.endSteel_pickaxe.name=EndstahlSpitzhacke
```
Comment (glowredman) [inline src/main/resources/assets/enderio/lang/de_DE.lang:787]:
```suggestion
item.endSteel_pickaxe.name=Endstahl-Spitzhacke
```

**Review by glowredman — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@Pilzinsel64 are you german nativ speaker ?

**Comment by Pilzinsel64** [comment]
@Dream-Master Hard to say "yes" in this case, but it's true. 🫢 Maybe it's because my left hand is broken and useless atm. But I don't want to look for a reason though, to be honest. Just didn't prof-read it a second time.
However, fixed all the suggestions.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**